README.md

```diff
@@ -3403,6 +3403,16 @@ Other Style Guides
     - We primarily use [`mocha`](https://www.npmjs.com/package/mocha) at Airbnb. [`tape`](https://www.npmjs.com/package/tape) is also used occasionally for small, separate modules.
     - 100% test coverage is a good goal to strive for, even if it’s not always practical to reach it.
     - Whenever you fix a bug, _write a regression test_. A bug fixed without a regression test is almost certainly going to break again in the future.
+    
+  <a name="testing--is-documentation"></a>
+  - [30.3](#testing--is-documentation) **Is your documentation**:
+    - Tests should explain what your code does.
+    - First write a case for each main functionality, and then their exceptions.
+    - Refactor tests until they are easy to read and understand.
+    - Never use _"private"_ properties to test code.
+    - Code inside tests should be copy-pasteable code to make new developments.
```

Comment:
i'm afraid i'm still not clear on what this one means :-/

Reply:
The idea is the following: because testing code is documentation, when you start writing new code, you have a very high probability of having that code already implemented in a test. So when someone starts coding and needs to use an existing code, he just have to go the specs, and make copy-paste. 
I liked the copy-pasteable expression because it implies that the testing code should be as close as possible to the production code, so each test initialisation, operation and check should be closer to other code initialisation, operation, and querying.
Although this idea, may be it should be clearer something like:

    - When someone starts writing code will 
       tests are the first place to know how to use existing code,
       make them easy to reuse code

Reply:
I'm still not clear on this. Are you saying that individual tests should be good starting points for adding new tests? I'm not sure this is even broadly true.